extensions/cornerstone/src/index.tsx

```diff
@@ -150,4 +161,19 @@ const cornerstoneExtension: Types.Extensions.Extension = {
   },
 };
 
+// TODO update upstream. It seems there is a new way of exporting modules.
+const extensionUtils = {
+  getSOPInstanceAttributes,
+  getEnabledElement,
+  annotation: {
+    setAnnotationColor,
+    setAnnotationLineDash,
+    getAnnotationColor,
+    setAnnotationSelected,
+    setAnnotationVisibility,
+    isAnnotationSelected,
+  },
+};
```

Comment:
why we are exporting these like this instead of `getUtilityModule`? What is the difference

Reply:
If they are exported like that, then they are accessible directly with correct typing.  The getUtilityModule export works, but doesn't take advantage of direct linking to things.  Some items should be exported direct, and others in getUtilityModule.